packages/react/src/components/AppShell/CHANGELOG.md

```
@@ -2,6 +2,13 @@
 
 The AppShell component is the main outer frame of an application. It provides the basic architecture to build an application inside of our admin.
 
+## 2026-06-25 `1.10.0`
+
+#### 🎉 New features
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)